frontend_lib/src/component/Timeline/EditCommentPopup.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'react-i18next'
+import i18n from '../../i18n.js'
+import { TracimComponent } from '../../tracimComponent.js'
+import { appContentFactory } from '../../appContentFactory.js'
+import { CUSTOM_EVENT } from '../../customEvent.js'
+import CardPopup from '../CardPopup/CardPopup.jsx'
+import IconButton from '../Button/IconButton.jsx'
+import CommentTextArea from './CommentTextArea.jsx'
+
+// require('./EditCommentPopup.styl')
```

Comment:
Is this normal?

Reply:
yes it is. In frontend_lib we don't require styl

See `rg require | rg styl` in `frontend_lib` and also #1156

Reply:
Yes, I'll add a comment